core/market/decentralized/src/matcher/error.rs

```diff
@@ -69,13 +74,13 @@ pub enum ModifyOfferError {
     #[error("Offer [{0}] not found.")]
     NotFound(SubscriptionId),
     #[error("Offer [{0}] already unsubscribed.")]
-    Unsubscribed(SubscriptionId),
+    AlreadyUnsubscribed(SubscriptionId),
     #[error("Can't unsubscribe expired Offer [{0}].")]
     Expired(SubscriptionId),
     #[error("Failed to unsubscribe Offer [{1}]. Error: {0}")]
-    UnsubscribeError(DbError, SubscriptionId),
+    Unsubscribe(DbError, SubscriptionId),
```

Comment:
It's confusing now. Maybe `UnsubscribeFailed` if you don't want error.

Reply:
This is part of the `Error`-named enum. It is obvious that it failed. It will be used as `...Error: Unsubscribe`